test/client/spec/components/victory-pie.spec.js

```diff
@@ -152,6 +152,20 @@ describe("components/victory-pie", () => {
 
       expect(xValues).to.eql([0, 1, 2, 3, 4, 5, 6, 7, 8]);
     });
+
+    it("renders data values sorted by sortKey prop and sortOrder", () => {
+      const data = range(9).map((i) => ({ x: i, y: i }));
+
+      const wrapper = shallow(
+        <VictoryPie data={data} sortKey={"x"} sortOder={"descending"}/>
```

Comment:
typo `sortOder` -> `sortOrder`

Reply:
@naveenblipp I couldn't figure out why CI was still failing after releasing your change in `victory-core`. Pesky typos. Approved pending this change.